modules/virt-editing-cdi-cpu-and-memory-defaults.adoc

```diff
@@ -3,54 +3,50 @@
 // * virt/virtual_machines/virtual_disks/virt-configuring-cdi-for-namespace-resourcequota.adoc
 
 [id="virt-editing-cdi-cpu-and-memory-defaults_{context}"]
-= Editing the `CDIConfig` object to override CPU and memory defaults
+= Editing the `CDI` object to override CPU and memory defaults
```

Comment:
I recently learned that we aren't supposed to use markup in headings, so I'd nix the backticks here.